apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2Test.java

```diff
@@ -769,77 +771,59 @@ public void createBundleWithoutPagingWithASizeOf2() throws IOException {
+        parsedRecords.stream().filter(r -> r instanceof HHAClaim).map(r -> r).findFirst().get();
 
-    claim.setLastUpdated(Instant.now());
+    ((HHAClaim) hhaClaim).setLastUpdated(Instant.now());
 
     FhirContext fhirContext = FhirContext.forR4();
-    HHAClaimTransformerV2 hhaClaimTransformer =
+    ClaimTransformerInterfaceV2 claimTransformerInterface =
         new HHAClaimTransformerV2(new MetricRegistry(), new NPIOrgLookup());
-    ExplanationOfBenefit genEob = hhaClaimTransformer.transform(claim);
+    ExplanationOfBenefit genEob = claimTransformerInterface.transform(hhaClaim);
     IParser parser = fhirContext.newJsonParser();
     String json = parser.encodeResourceToString(genEob);
     List<IBaseResource> eobs = new ArrayList<IBaseResource>();
     eobs.add(parser.parseResource(ExplanationOfBenefit.class, json));
 
-    HospiceClaim hospiceClaim =
-        parsedRecords.stream()
-            .filter(r -> r instanceof HospiceClaim)
-            .map(r -> (HospiceClaim) r)
-            .findFirst()
-            .get();
+    Object hospiceClaim =
+        parsedRecords.stream().filter(r -> r instanceof HospiceClaim).map(r -> r).findFirst().get();
 
-    claim.setLastUpdated(Instant.now());
+    ((HospiceClaim) hospiceClaim).setLastUpdated(Instant.now());
 
-    fhirContext = FhirContext.forR4();
-    HospiceClaimTransformerV2 hospiceClaimTransformerV2 =
+    claimTransformerInterface =
         new HospiceClaimTransformerV2(new MetricRegistry(), new NPIOrgLookup());
-    genEob = hospiceClaimTransformerV2.transform(hospiceClaim);
+    genEob = claimTransformerInterface.transform(hospiceClaim);
     parser = fhirContext.newJsonParser();
     json = parser.encodeResourceToString(genEob);
-
     eobs.add(parser.parseResource(ExplanationOfBenefit.class, json));
 
-    DMEClaim dmeClaim =
-        parsedRecords.stream()
-            .filter(r -> r instanceof DMEClaim)
-            .map(r -> (DMEClaim) r)
-            .findFirst()
-            .get();
+    Object dmeClaim =
+        parsedRecords.stream().filter(r -> r instanceof DMEClaim).map(r -> r).findFirst().get();
 
-    claim.setLastUpdated(Instant.now());
+    ((DMEClaim) dmeClaim).setLastUpdated(Instant.now());
 
-    fhirContext = FhirContext.forR4();
-    DMEClaimTransformerV2 dmeClaimTransformerV2 =
+    claimTransformerInterface =
         new DMEClaimTransformerV2(
             new MetricRegistry(), FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting());
-    genEob = dmeClaimTransformerV2.transform(dmeClaim, false);
+    genEob = claimTransformerInterface.transform(dmeClaim, false);
     parser = fhirContext.newJsonParser();
     json = parser.encodeResourceToString(genEob);
-
     eobs.add(parser.parseResource(ExplanationOfBenefit.class, json));
 
-    InpatientClaim inpatientClaim =
+    Object inpatientClaim =
         parsedRecords.stream()
             .filter(r -> r instanceof InpatientClaim)
-            .map(r -> (InpatientClaim) r)
+            .map(r -> r)
```

Comment:
This map does nothing.  This would eliminate the `Object` and explicit cast:

```
    InpatientClaim inpatientClaim =
        parsedRecords.stream()
            .filter(r -> r instanceof InpatientClaim)
            .map(InpatientClaim.class::cast)
            .findFirst()
            .get();

    inpatientClaim.setLastUpdated(Instant.now());
```

Reply:
implemented static cast throughout.